### PR TITLE
add zoom message support

### PR DESCRIPTION
**Pull request by ericmigi**
now shows zoom call messages

### CODE REVIEW

**Comment by Sorunome** [comment]
Is that a normal, documented block?

File: src/slackmessageparser.ts [hunk 1]
```diff
@@ -387,6 +387,15 @@ export class SlackBlocksParser {
 				retStr += "</p>";
 				return retStr;
 			}
+			case "call": {
+				const call = block as ISlackBlockCall;
+				if (call.call && call.call.v1) {
+					const { name, join_url } = call.call.v1;
+					const content = name ? name : escapeHtml(join_url);
+					return `<a href="${join_url}">${content}</a>`;
```
Comment (Sorunome) [inline src/slackmessageparser.ts:395]: please `escapeHtml()` your variables ^^ name and join_url aren't neecessarily escaped

**Review by Sorunome — CHANGES_REQUESTED** [review]
Otherwise looks good!

**Comment by ericmigi** [comment]
looks good, thanks for reviewing!

**Comment by ericmigi** [comment]
anything else I need to do?

**Comment by Sorunome** [comment]
fix the requested changes? (maybe you forgot to push?)

Am 12. Mai 2020 20:26:07 MESZ schrieb Eric Migicovsky <notifications@github.com>:

**Pull request closed without merging**